Makefile

```diff
@@ -13,9 +13,16 @@ MIN_GOLANG_VERSION=1.16 # for building ghpc
 ENG = ./cmd/... ./pkg/...
 TERRAFORM_FOLDERS=$(shell find ./modules ./community/modules ./tools -type f -name "*.tf" -not -path '*/\.*' -exec dirname "{}" \; | sort -u)
 PACKER_FOLDERS=$(shell find ./modules ./community/modules ./tools -type f -name "*.pkr.hcl" -not -path '*/\.*' -exec dirname "{}" \; | sort -u)
+
+ifneq (, $(shell which git))
+## GIT IS PRESENT
+ifneq (,$(wildcard .git))
+## GIT DIRECTORY EXISTS
```

Comment:
This LGTM but I am not expert with shell/make syntax. @cboneti , could you give the syntax a once over.

Reply:
Second thought, I see this syntax used elsewhere in the make file. I am good to approve.